TASVideos/Pages/Shared/_MovieModule.cshtml

```diff
@@ -38,149 +47,152 @@
 }
 <card class="border border-primary">
 	<cardheader class="bg-cardprimary p-2">
-        <row class="gx-3">
-            <div class="pixelart-image col-auto pe-0" condition="!string.IsNullOrWhiteSpace(Model.TierIconPath)">
-                <img src="/@Model.TierIconPath" />
-            </div>
-            <h4 class="col m-0"><a asp-page="View" asp-route-id="@Model.Id" class="text-decoration-none"><span class="text-dark">@Model.Title</span></a></h4>
-            <div class="col-auto ps-0">
-                @foreach (var flag in Model.Flags.Where(f => !string.IsNullOrWhiteSpace(f.IconPath)))
-                {
-                    <a href="@flag.LinkPath">
-                        <img class="pixelart-image" title="@flag.Name" alt="[@flag.Name]" src="/@flag.IconPath" />
-                    </a>
-                }
-            </div>
-        </row>
+		<row class="gx-3">
+			<div class="col-auto pe-0" condition="!string.IsNullOrWhiteSpace(Model.TierIconPath)">
+				<img style="width: 18px" src="/@tierIconPath2x"
+					 srcset="/@Model.TierIconPath .5x,
+					 /@tierIconPath2x 1x,
+					 /@tierIconPath4x 2x" />
+			</div>
+			<h4 class="col m-0"><a asp-page="View" asp-route-id="@Model.Id" class="text-decoration-none"><span class="text-dark">@Model.Title</span></a></h4>
+			<div class="col-auto ps-0">
+				@foreach (var flag in Model.Flags.Where(f => !string.IsNullOrWhiteSpace(f.IconPath)))
+				{
+					<a href="@flag.LinkPath">
+						<img class="pixelart-image" title="@flag.Name" alt="[@flag.Name]" src="/@flag.IconPath" />
+					</a>
+				}
+			</div>
+		</row>
 	</cardheader>
-    <cardbody class="px-2 py-0">
```

Comment:
What happened here? whitespace?

Reply:
yeah you want everything standardized on tab delimited right?

Reply:
Yes please.  If there is a way to specify tabs in .cshtml files in .editorconfig, that would be ideal, so there is no need to nag